README.md

```diff
@@ -1 +1,159 @@
+- [📝 License](#license)
+
+# 📖 [My-Portfolio] <a name="setting Up a linters"></a>
+
+**[My-Portfolio]** is a project to set up html-css linters in order to be able to detect bugs in our code
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. HTML
+2. CSS
+3. FontAwesome
+4. LINTERS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">html</a></li>
+    <li><a href="https://reactjs.org/">css</a></li>
+    <li><a href="https://reactjs.org/">javascipt </a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+Added html.index file
+Added style.css
+Added linters.yml
+
+- **[html.index]**
+- **[style.css]**
+- **[linters.yml]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- web browser (google chrome)
+- code editor
+- git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+    - https://github.com/Kal2403/my-portfolio-project.git
+
+### Install
+
+Install this project with:
+
+    1. Clone this repository to your local machine:
+
+        - git clone https://github.com/Kal2403/my-portfolio-project.git
+
+    2. Open the `index.html` file in your favorite web browser.
+
+### Usage
+
+Once you have opened the `index.html` file in your browser, you should see two section's.
+- First one is the header with logo and a menu bar to desplay. You will see portfolio, about me and contact.
+- Second you will the section main who has the content of me and icons from my personal social medias.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 ** Cristhian Castillo **
+
+- GitHub: [@githubhandle](https://github.com/Kal2403)
+- Twitter: [@twitterhandle](https://twitter.com/Kal24031992)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cristhian-castillo-41136322a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add header
+- [ ] Add style
+- [ ] Add more styling
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give me a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the opportunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
```

Comment:
- [ ] You can remove the FAQ section it's the only optional section or fill it with the correct contents. That will make your README file more professional and clean.